docs/faq.md

```diff
@@ -34,6 +34,20 @@ Most developer should not need to worry about these and can just use the `api.tr
 
 ---
 
+## Health check
+
+> How can I check whether the service is working?
+
+You can to that by quering our [`regions.json`](/#tag/Configuration%2Fpaths%2F~1regions.json%2Fpost) endpoint, adding the `X-TripGo-HealthCheck` heather:
```

Comment:
```suggestion
You can to that by querying the [`regions.json`](/#tag/Configuration%2Fpaths%2F~1regions.json%2Fpost) endpoint, adding the `X-TripGo-HealthCheck` header:
```

What is the expected response when it's working? Just an empty 200 OK? Is the body in the request necessary or can it be omitted?